.gitignore

```diff
@@ -22,6 +22,7 @@ actual.json
 console_output.txt
 expected.json
 run_mode_metadata.json
+https:/**
```

Comment:
This doesn't hurt, but wondering if this is an actual directory we need to ignore here?

Reply:
I thought we did at one point....but I don't think we do anymore.